### ISSUE TITLE
Problem with label generated for sections with inline math expressions

### ISSUE THREAD

**Issue by juanfcocontreras** [JelteF/PyLaTeX#224]
Minimal example:

```
title = 'This is an example $tetha$'
with doc.create(Section(NoEscape(title))):
    doc.append('Some regular text and some')
```

The problem is with the label generated, it includes the `$` symbol. If `label=False`, or if I remove `$` from the label, the issue is fixed:

```
def clean_label(label):
    return NoEscape(label).replace('$', '')

title = 'This is an example $tetha$'
with doc.create(Section(NoEscape(title), label=clean_label(title))):
    doc.append('Some regular text and some')
```

I think it may be get fixed by adding `$` to `_remove_invalid_char` function in `labelref.py`, but I'm not totally sure.

**Comment by JelteF** [comment]
Yeah that makes sense. Feel free to create a PR.

**Comment by juanfcocontreras** [comment]
Done! Thanks!

**Comment by jeffheaton** [comment]
This also happens if a subsection has an ampersand (&) inside of a section/subsection. 

Also related:  https://stackoverflow.com/questions/52721270/pylatex-special-characters-in-subsection

**Comment by JelteF** [comment]
Just released v1.3.1 that should fix this issue
